tests/datatypes/test_set.py

```diff
@@ -62,6 +62,34 @@ def test_iter(self):
             """)
 
 
+    def test__ior__(self):
+        self.assertCodeExecution("""
+        a = {'a', 'b', 'c'}
+        b = {'a', 'c', 'd'}
+        a.__ior__(b)
+        """)
+
+    def test__iand__(self):
+        self.assertCodeExecution("""
+        a = {'a', 'b', 'c'}
+        b = {'a', 'c', 'd'}
+        a.__iand__(b)
+        """)
+
+
+    def test_union(self):
+        self.assertCodeExecution("""
+        a = {'a', 'b', 'c'}
+        b = {'a', 'c', 'd'}
+        a.union(b)        
```

Comment:
Although this is executes the right code, it won't actually test anything. `assertCodeExecution()` performs a comparison of the output of executed code - so, this test will verify that the code doesn't output anything. This will verify that the code runs without error, but won't verify that the output is actually correct.

To test that the output is correct, you have to print the result of the operation so that there is something to compare.